storage/etcd/config.go

```diff
@@ -0,0 +1,82 @@
+package etcd
+
+import (
+	"time"
+
+	"github.com/coreos/dex/storage"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
+	"github.com/sirupsen/logrus"
+)
+
+type EtcdSSL struct {
+	ServerName string `json:"server_name" yaml:"server_name"`
+	CAFile     string `json:"ca_file" yaml:"ca_file"`
+	KeyFile    string `json:"key_file" yaml:"key_file"`
+	CertFile   string `json:"cert_file" yaml:"cert_file"`
+}
+
+type Etcd struct {
+	Endpoints        []string `json:"endpoints" yaml:"endpoints"`
+	DialTimeout      int      `json:"dial_timeout" yaml:"dial_timeout"`
+	AutoSyncInterval int      `json:"auto_sync_interval" yaml:"auto_sync_interval"`
+	Username         string   `json:"username" yaml:"username"`
+	Password         string   `json:"password" yaml:"password"`
+	SSL              EtcdSSL  `json:"ssl" yaml:"ssl"`
+	RejectOldCluster bool     `json:"reject-old-cluster" yaml:"reject-old-cluster"`
```

Comment:
What does this do?

Reply:
```
// RejectOldCluster when set will refuse to create a client against an outdated cluster.
RejectOldCluster bool `json:"reject-old-cluster"`
```

as before, will also remove this knob for now.